src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -177,3 +173,111 @@
     }
   }
 }
+
+.cd-navbar-primary > li > a {
+  border: 0;
+}
+
+.cd-navbar-primary .active > a,
+.cd-navbar-primary > .active > a:focus,
+.cd-navbar-primary > .active > a:hover {
+  color: $color-nav-links;
+  background-color: $color-nav-links-hover;
+  border: 0;
+}
+/* ---------------------------------------------------
+    SIDEBAR STYLE
+----------------------------------------------------- */
+
+$sidebar-width: 200px;
+
+.wrapper {
+  display: flex;
+  width: 100%;
+}
+
+#sidebar {
+  width: $sidebar-width;
+  margin-top: 41px;
+  padding-top: 10px;
+  background: $color-navbar-bg;
+  overflow-y: auto;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  color: #fff;
+  transition: all 0.3s;
+}
+
+#sidebar.active {
+  margin-left: -$sidebar-width;
+}
+
+#sidebar ul.components {
+  padding: 20px 0;
+  border-bottom: 1px solid #47748b;
```

Comment:
You should remove this line because if creates an ugly line.
![Bildschirmfoto vom 2019-11-29 14-06-22](https://user-images.githubusercontent.com/1897962/69870911-9fef6880-12b1-11ea-88fa-3e000786ad40.png)

```suggestion
```

Reply:
Done.